src/main/java/io/codearte/jfairy/producer/company/CompanyProvider.java

```diff
@@ -67,8 +68,27 @@ private void generateDomain() {
 		if (domain != null) {
 			return;
 		}
-		domain = TextUtils.stripAccents(StringUtils.strip(StringUtils.deleteWhitespace(name.toLowerCase()), ".").replace("/", ""))
-				+ "." + dataMaster.getRandomValue(DOMAIN);
+//		domain = TextUtils.stripAccents(StringUtils.strip(StringUtils.deleteWhitespace(name.toLowerCase()), ".").replace("/", ""))
```

Comment:
Maybe migration to Java 8 and a default method in that interface would be useful to reduce amount of changes in other providers?